.github/workflows/smoketest.yml

```diff
@@ -227,19 +232,14 @@ jobs:
           fi
         fi
 
-    - name: Dependencies20.04
+    - name: Dependencies22.04
       if: ${{ steps.check-cache.outputs.state != '' }} # if 'state' set then step needed
       run: |
         sudo apt-get update
-        sudo apt-get -yq install bison flex build-essential binutils-dev libldap2-dev libcppunit-dev libicu-dev libxslt1-dev \
-          zlib1g-dev libboost-regex-dev libarchive-dev libv8-dev default-jdk libapr1-dev libaprutil1-dev libiberty-dev \
-          libhiredis-dev libtbb-dev libxalan-c-dev libnuma-dev libevent-dev libatlas-base-dev libblas-dev python3-dev \
-          default-libmysqlclient-dev libsqlite3-dev libmemcached-dev libcurl4-openssl-dev pkg-config libtool autotools-dev automake \
-          libssl-dev xmlstarlet
-
-        curl https://deb.nodesource.com/gpgkey/nodesource.gpg.key | sudo apt-key add -
-        sudo apt-add-repository "deb https://deb.nodesource.com/node_16.x $(lsb_release -sc) main"
-        sudo apt-get update -y
+        sudo apt-get install -y ${{ env.OS_DEPENDENCIES }}
+        sudo apt-get install -y xmlstarlet
+
+        curl -fsSL https://deb.nodesource.com/setup_19.x | sudo -E bash -
```

Comment:
Lets keep this on 16 for now (as its what we are using everywhere else).  Note:  When we do upgrade it will be to 18